questions_pop_culture.py

```diff
@@ -15,6 +15,9 @@
     'In "Silicon Valley" Season 3 Episode 1, what language is being used?\n@SiliconHBO',  # https://www.reddit.com/r/SiliconValleyHBO/comments/4gbra7/i_got_the_silicon_valley_s03e01_code_to_compile/)
     'What day is celebrated as Programmer\'s day? Why?\n#ProgrammersDay',  #
     '🐠In Mr. Robot, what is the name of Elliot\'s betta fish? @whoismrrobot #MrRobot',  # Qwerty
+    'Who was the first Disney Princess? @DisneyStudios' #Disney
```

Comment:
This is a fun question, but is it code-related? 
Code Questions bot is meant to help developers learn cod things together.